sharqiyya-app-homepage/index.html

```diff
@@ -0,0 +1,557 @@
+              </div>
+
+              <div class="dates-bar__box">
+                <div class="dates-bar__box-top">21</div>
+                <div class="dates-bar__box-bottom">sat</div>
+              </div>
+
+              <img class="dates-bar__arrow" src="images/icons/forward-arrow-grey.svg">
+
+            </div>
+            <div class="dates-bar__right">
+              <img src="images/icons/search.svg">
+              <div class="dates-bar__search">Search for an event or activity</div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="homepage">
+        <div class="tags-bar">
+          <div class="tags-bar__tag tags-bar__tag_selected">
+            Everything
+          </div>
+          <div class="tags-bar__tag">
+            Halloween Special
+          </div>
+          <div class="tags-bar__tag">
+            Arts & Culture
+          </div>
+          <div class="tags-bar__tag">
+            Conferences & Workshops
+          </div>
+          <div class="tags-bar__tag">
+            Exhibitions
+          </div>
+          <div class="tags-bar__tag">
+            Sports & Outdoor
+          </div>
+          <div class="tags-bar__tag">
+            Food & Drinks
+          </div>
+          <div class="tags-bar__tag">
+            Nightlife
+          </div>
+          <div class="tags-bar__tag">
+            More
+          </div>
+        </div>
+
+        <div class="homepage-content">
+          <div class="homepage-content__title">
+            15 EVENTS FOUND ON SATURDAY AUGUST 9
+          </div>
+          <div class="homepage-content__events-list">
+
+
+            <div class="homepage-content__event">
+              <div class="homepage-content__event-like-button homepage-content__event-like-button_liked">
+                <img src="images/icons/heart-white.svg"/>
+              </div>
+              <img src="images/events/event-1.png" class="homepage-content__event-image" />
+              <div class="homepage-content__event-details">
+                <div class="homepage-content__from-to">
+                  JAN 11 TILL MAR 08
+                </div>
+                <div class="homepage-content__event-title">
+                  BIG CITY LIFE - Exhibition Opening
+                </div>
+                <div class="homepage-content__event-timing-and-location">
+                  <img class="homepage-content__icon" src="images/icons/time.svg">
+                  10:00 am till 11:00 pm
+                </div>
+                <div class="homepage-content__event-timing-and-location">
+                  <img class="homepage-content__icon" src="images/icons/location.svg">
+                  Salon Beyrouth restaurant, Mohammed Abdel Baki street, Beirut, Lebanon
+                </div>
+                <div class="homepage-content__event-tags">
+                  # Arts & Culture&nbsp;&nbsp;&nbsp;&nbsp;#Conferences & Workshops
```

Comment:
It's better to have each tag in its own div and css element, `homepage-content__event-tag` (same for all the event cards). It's also bad practice to depend on `&nbsp` for spacing, it should be done in css.